src/control/server/engine/config.go

```diff
@@ -343,6 +346,23 @@ func (c *Config) Validate() error {
 		return errors.Wrap(err, "validate engine log subsystems")
 	}
 
+	// ensure 18KB ABT stack size for an engine with DCPM storage class
+	if c.Storage.Tiers[0].Class == storage.ClassDcpm {
```

Comment:
Would be great to have new unit test cases for this new block of logic. I see a few cases here:
- env var not set with SCM (OK, env vars list updated)
- env var not set without SCM (OK, no change)
- env var contains non-integer junk (error)
- stack size too small with SCM (error)
- smaller stack size without SCM (OK)

It may be easiest to test by breaking the if-block out into a helper function and writing tests for that, rather than updating an existing unit test to check whether the Config was updated. Either way would be acceptable.

Reply:
So far all existing tests have been updated.